lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -1,16 +1,26 @@
+require "cc/engine/analyzers/parser_error"
+
 module CC
   module Engine
     module Analyzers
       class Base
+        RESCUABLE_ERRORS = [
```

Comment:
Opinion question for the team: should we also catch `Timeout::Error`? Flay does internally (not in a code path we're using, FWIW). I think the thinking behind it was that if the parser took too long on a file, it's probably a parser bug (or maybe an absurdly large file). We were implicitly catching it before.